# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

TranslateCommandActionGoal action_goal
TranslateCommandActionResult action_result
TranslateCommandActionFeedback action_feedback
